### PR TITLE
fix bug for sending documents and bug in ins search

### LINKED ISSUE #5014: send document error
### Describe the bug

from the forum:
https://community.open-emr.org/t/document-notes-and-email-error/17812/2?u=stephenwaite

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #5014 #5017

#### Short description of what this resolves:
https://github.com/openemr/openemr/pull/4435#discussion_r643380507

#### Changes proposed in this pull request:
add missing require,

### CODE REVIEW

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
2DV-2

**Comment by bradymiller** [comment]
btw, there is now a button on the automated testing in github to only retry the failed ci test (so it doesn't need to do them all anymore) :)

File: src/Services/AddressService.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 use Particle\Validator\Validator;
 
-class AddressService
+class AddressService extends BaseService
```
Comment (sjpadgett) [inline src/Services/AddressService.php:19]: Nice job picking up on the base service.

**Review by sjpadgett — APPROVED** [review]

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
looks good. would also bring into rel-610 branch.

**Pull request merged**